dash_bio/Ideogram.py

```diff
@@ -14,15 +14,16 @@ class Ideogram(Component):
 by side tracks to show important genomic data.
 
 Go here to see it in action: https://eweitz.github.io/ideogram/
+Reference of the component's props :
+https://github.com/eweitz/ideogram/blob/
+770ce1fe5ae4f27dd435f9f99948118af88f8349/api.md
```

Comment:
It doesn't look like you regenerated file `Ideogram.py` after commit 0a7c4dc87c1929c8f.